libs/payments/cart/src/lib/cart.utils.ts

```diff
@@ -26,3 +57,10 @@ export function stripeErrorToErrorReasonId(
       return CartErrorReasonId.Unknown;
   }
 }
+
+export function handleEligibilityStatus(eligibility: EligibilityStatus) {
+  return (
+    CartEligibilityDetailsMap[eligibility] ||
+    CartEligibilityDetailsMap[EligibilityStatus.INVALID]
```

Comment:
The second condition here isn't possible, right?

Reply:
Will update!